packages/react-vapor/src/components/actionable-item/styles/ActionableItem.scss

```
@@ -14,12 +14,14 @@ $actionable-item-dots-width: 12px;
 }
 
 .actionableItemContent {
-    padding: $button-padding-y $button-padding-x;
+    padding: var(--button-no-tag-button-padding-y) var(--button-padding-x) var(--button-no-tag-button-padding-y)
+        var(--button-padding-x);

```

Can be simplified since you use the same var twice

```suggestion
    padding: var(--button-no-tag-button-padding-y) var(--button-padding-x);
```